src/main/java/nl/rijksoverheid/mev/gezagsmodule/domain/gezagvraag/OudersOverledenOfOnbevoegdTotGezag.java

```diff
@@ -24,13 +36,20 @@ protected OudersOverledenOfOnbevoegdTotGezag(final GezagsBepaling gezagsBepaling
     @Override
     public void perform() {
         preconditieCheckOudersGeregistreerd();
-        boolean ouder1OverledenOfOnbevoegd = gezagsBepaling.getPlOuder1().isOverledenOfOnbevoegd();
-        boolean ouder2OverledenOfOnbevoegd = gezagsBepaling.getPlOuder2().isOverledenOfOnbevoegd();
-        if (ouder1OverledenOfOnbevoegd && ouder2OverledenOfOnbevoegd) {
-            answer = V4A_2_JA_BEIDEN;
-        } else if (ouder1OverledenOfOnbevoegd) {
+
+        var optionalIsOuder1OverledenOfOnbevoegdToken = gezagsBepaling.getPlOuder1().isOverledenOfOnbevoegdEncoded();
+        var optionalIsOuder2OverledenOfOnbevoegdToken = gezagsBepaling.getPlOuder2().isOverledenOfOnbevoegdEncoded();
+        var isOuder1OverledenOfOnbevoegd = optionalIsOuder1OverledenOfOnbevoegdToken.isPresent();
+        var isOuder2OverledenOfOnbevoegd = optionalIsOuder2OverledenOfOnbevoegdToken.isPresent();
+
+        if (isOuder1OverledenOfOnbevoegd && isOuder2OverledenOfOnbevoegd) {
+            var isOuder1OverledenOfOnbevoegdToken = optionalIsOuder1OverledenOfOnbevoegdToken.get();
+            var isOuder2OverledenOfOnbevoegdToken = optionalIsOuder2OverledenOfOnbevoegdToken.get();
+            var key = "%c%c".formatted(isOuder1OverledenOfOnbevoegdToken, isOuder2OverledenOfOnbevoegdToken);
```

Comment:
zie eerdere comment

Reply:
Besproken in https://github.com/BRP-API/brp-api-gezag/pull/227#discussion_r1890146316.